src/env/log_fd.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2017-present, PingCAP, Inc. Licensed under Apache-2.0.
+
+#[cfg(any(target_os = "unix", target_os = "linux"))]
+mod nix;
+#[cfg(any(target_os = "unix", target_os = "linux"))]
+pub use nix::LogFd;
+
+#[cfg(not(any(target_os = "unix", target_os = "linux")))]
+mod plain;
```

Comment:
I think s/plain/std is okay, consistent with nix.

Reply:
Was named `mod std` until rust complained about naming clash with the its own "std".